apps/docs/content/components/PricingOptions.mdx

```diff
@@ -0,0 +1,747 @@
+---
+title: Pricing options
+status: Experimental
+a11yReviewed: false
+source: https://github.com/primer/brand/blob/main/packages/react/src/PricingOptions/PricingOptions.tsx
+storybook: '/brand/storybook/?path=/story/components-pricingoptions--default'
+description: Use the pricing options to display pricing plan information on a page.
+---
+
+import {Label} from '@primer/react'
+import {StarIcon, CopilotIcon} from '@primer/octicons-react'
+
+import {PropTableValues} from '../../src/components'
+import {
+  EyebrowBannerIconColors,
+  EyebrowBannerLabelColors,
+} from '@primer/react-brand'
+
+```js
+import {EyebrowBanner} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default presentation
+
+```jsx live
+<PricingOptions>
+  <PricingOptions.Item>
+    <PricingOptions.Label>Recommended</PricingOptions.Label>
+    <PricingOptions.Heading>Copilot</PricingOptions.Heading>
+    <PricingOptions.Description>
+      Copilot in the coding environment.
+    </PricingOptions.Description>
+    <PricingOptions.Price
+      currencySymbol="$"
+      trailingText="per month / $100 per year"
+    >
+      10
+    </PricingOptions.Price>
+    <PricingOptions.FeatureList>
+      <PricingOptions.FeatureListItem>
+        Everything in Copilot Business plus:
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Chat in IDE and Mobile
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        CLI assistance
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Code completions
+      </PricingOptions.FeatureListItem>
+    </PricingOptions.FeatureList>
+    <PricingOptions.PrimaryAction href="/buy">
+      Buy now
+    </PricingOptions.PrimaryAction>
+    <PricingOptions.SecondaryAction href="/contact">
+      Contact sales
+    </PricingOptions.SecondaryAction>
+  </PricingOptions.Item>
+
+  <PricingOptions.Item>
+    <PricingOptions.Label>Recommended</PricingOptions.Label>
+    <PricingOptions.Heading>Copilot Business</PricingOptions.Heading>
+    <PricingOptions.Description>
+      Copilot personalized to your organization throughout the software
+      development lifecycle. Requires GitHub Enterprise Cloud.
+    </PricingOptions.Description>
+    <PricingOptions.Price currencySymbol="$" trailingText="per user / month">
+      39
+    </PricingOptions.Price>
+    <PricingOptions.FeatureList>
+      <PricingOptions.FeatureListItem>
+        Everything in Copilot Business plus:
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Chat in IDE and Mobile
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        CLI assistance
+      </PricingOptions.FeatureListItem>
+      <PricingOptions.FeatureListItem>
+        Code completions
+      </PricingOptions.FeatureListItem>
+    </PricingOptions.FeatureList>
+    <PricingOptions.PrimaryAction href="/buy">
```

Comment:
General comment about the `href` value for all examples. They will all need to be updated to use `as="a"`, because they default to `button` elements and href isn't forwarded correctly. While doing that, can these also be made `#` so they don't 404 when actual links please?